examples/patterns/navigation/subnav.html

```diff
@@ -0,0 +1,167 @@
+    </form>
+    <span class="u-off-screen">
+      <a href="#main-content">Jump to main content</a>
+    </span>
+    <ul class="p-navigation__links" role="menu">
+      <li class="p-navigation__link p-subnav__toggle" role="menuitem" id="link-1">
+        <a href="#link-1-menu" class="p-subnav__toggle-link">
+          Link 1 <i class="p-subnav__toggle-indicator"></i>
+        </a>
+        <ul class="p-subnav u-hide" id="link-1-menu" aria-hidden="true">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav__toggle" role="menuitem" id="link-2">
+        <a href="#link-2-menu" class="p-subnav__toggle-link">
+          Link 2 <i class="p-subnav__toggle-indicator"></i>
+        </a>
+        <ul class="p-subnav u-hide" id="link-2-menu" aria-hidden="true">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 4</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 5</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav__toggle" role="menuitem" id="link-3">
+        <a href="#link-3-menu" class="p-subnav__toggle-link">
+          Link 3 <i class="p-subnav__toggle-indicator"></i>
+        </a>
+        <ul class="p-subnav u-hide" id="link-3-menu" aria-hidden="true">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </nav>
+</header>
+
+<script>
+var dropdowns = document.querySelectorAll('.p-subnav');
+var dropdownToggles = document.querySelectorAll('.p-subnav__toggle');
+var dropdownToggleLinks = document.querySelectorAll('.p-subnav__toggle-link');
+
+dropdownToggleLinks.forEach(function(dropdownToggleLink) {
+  dropdownToggleLink.addEventListener('click', function(event) {
+    event.preventDefault();
+  });
+});
+
+dropdownToggles.forEach(function(dropdownToggle) {
+  dropdownToggle.addEventListener('click', function(event) {
+    event.stopPropagation();
+
+    var clickedDropdown = this;
+
+    dropdownToggles.forEach(function(dropdownToggle) {
+      var dropdownContent = document.getElementById(dropdownToggle.id + "-menu");
```

Comment:
I dont think we need to tie the id and classes together because we have them contained together. We should just do something like toggle + menu == open.

Reply:
I'm not certain that I follow. The classes are used for styling and the ID is used for semantic linking of the elements. Any agent that parses the raw HTML will still get the semantic purpose of the toggle to link to the subnav.

Reply:
I see what you mean for simplifying the JS. I'll restructure the selectors slightly and see if that's better.

Reply:
Now the HTML is restructured, the JS now uses classes rather than the ID to select the correct menu.